Improve discoverability of LangSmith Sandboxes in docs search and navigation (#4339)
langchain-ai/docs: src/oss/deepagents/sandboxes.mdx

    **Using Deep Agents Code?** Deep Agents Code has built-in sandbox support via the `--sandbox` flag. See [Use remote sandboxes](/oss/deepagents/code/remote-sandboxes) for Deep Agents Code-specific setup, flags (`--sandbox-id`, `--sandbox-setup`), and examples.
</Tip>

<Note>

>     **Looking for LangSmith Sandboxes?** LangSmith provides first-party managed sandboxes you can use directly from the LangSmith UI or SDK — no third-party account required. See [LangSmith Sandboxes](/langsmith/sandboxes) for managed sandbox resources, snapshots, service URLs, and the auth proxy.

Reviewer: ```suggestion
**If you're looking for LangSmith sandboxes:** LangSmith provides first-party managed sandboxes you can use directly from the LangSmith UI or SDK without a third-party account required. For managed sandbox resources, snapshots, service URLs, and the auth proxy, refer to [LangSmith Sandboxes](/langsmith/sandboxes).
```
